server/container/guice/guice-common/src/main/java/org/apache/james/utils/ExtendedServerProbe.java

```diff
@@ -37,4 +39,8 @@ void appendMessage(String username, MailboxPath mailboxPath, InputStream message
             throws BadCredentialsException, MailboxException;
 
     Mailbox<Id> getMailbox(String namespace, String user, String name);
+
+    void modifyVacation(AccountId accountId, Vacation vacation);
```

Comment:
I understand the need for that, but this _Probe_ thing is becoming a mess. We should start dividing different domains into different classes.